_includes/sidebar.html

```diff
@@ -33,9 +33,12 @@
                     <div class="hr"></div>
                     <div class="widget">
                         <div class="description text-justify">
-                            We are an open source web framework for PHP delivered as
-                            a C extension offering high performance and lower
-                            resource consumption
+                            We're a nonprofit organization that makes solutions for web devlopers.
+                            Or products is <a href="https://phalconphp.com/en/" target="_blank">Phalcon</a>,
+                            <a href="https://zephir-lang.com/en" target="_blank">Zephir</a> and
+                            <a href="https://github.com/phalcon?type=source" target="_blank">others</a>.
+                            If you would like to help us stay free and open further,
+                            please consider <a href="https://phalcon.link/fund" arget="_blank">making a small donation</a>.
```

Comment:
We're a nonprofit organization that makes creates for web developers.
Or projects are <a href="https://phalconphp.com/en/" target="_blank">Phalcon</a>,
<a href="https://zephir-lang.com/en" target="_blank">Zephir</a> and
<a href="https://github.com/phalcon?type=source" target="_blank">others</a>.
If you would like to help us stay free and open, please consider 
<a href="https://phalcon.link/fund" arget="_blank">supporting us</a>.